apps/tomcat/9.0.97/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nexus:
-    image: bitnami/tomcat:9.0.97
+    image: bitnami/tomcat:9.0.98
     restart: always
     container_name: ${CONTAINER_NAME}
     environment:
```

Comment:
The provided Git patch indicates an update to the `bitnami/tomcat` Docker image version from 9.0.97 to 9.0.98 for your Nexus service. There are no apparent irregularities, potential issues, or major optimizations suggested in this change. The only notable element is that you've updated the tag and likely followed it with more configuration details not shown in the snippet above. If there were specific changes in behavior between versions 9.0.97 and 9.0.98 related to security updates, performance improvements, or bug fixes, they would be reflected further upstream in the commit history of that repository.